services/blockchain-indexer/shared/dataService/knownAccounts.js

```diff
@@ -0,0 +1,80 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const { HTTP, Logger } = require('lisk-service-framework');
+const util = require('util');
+
+const config = require('../../config');
+const { LENGTH_NETWORK_ID } = require('../constants');
+const { getNetworkStatus } = require('./business/network');
+
+const logger = Logger();
+
+const networkConfig = Object.values(config.networks);
+const staticUrl = config.endpoints.liskStatic;
+
+let knowledge = {};
+
+const getAccountKnowledge = (address) => {
+	if (knowledge[address]) return knowledge[address];
+	return {};
+};
+
+const resolveNetworkByChainID = (chainID) => {
+	const networkID = chainID.substring(0, LENGTH_NETWORK_ID);
+
+	const matchingNetwork = networkConfig.flatMap((network) => network)
+		.filter((ntwk) => ntwk.chainID.startsWith(networkID)).find(Boolean);
+
+	return matchingNetwork ? matchingNetwork.name : null;
+};
+
+const reloadAccountKnowledge = async () => {
+	logger.debug('Reloading known accounts...');
+
+	try {
+		const { data: { chainID } } = await getNetworkStatus();
+		const networkName = resolveNetworkByChainID(chainID);
+
+		if (networkName) {
+			const res = await HTTP.request(`${staticUrl}/known_${networkName}.json`);
```

Comment:
Would it be better to use the framework helper api instead?

https://github.com/LiskHQ/lisk-service/blob/9830625a9a25269b704ec474a7a8434334b29bec/framework/src/http.js#L108